apps/mobile/src/features/SetupWallet/useCases/CreateWallet/VerifyRecoveryPhraseScreen.tsx

```diff
@@ -345,6 +349,18 @@ const WordBadges = ({
 
   const {palette: p} = useTheme()
 
+  const styles = {
```

Comment:
is used only in one place migrate to the componennt

Reply:
moved to the component